### ISSUE TITLE
Data Issue:  

### ISSUE THREAD

**Issue by pnickelnm** [nytimes/covid-19-data#411]
### Describe the issue:
- [ ] Incorrect number of cases or deaths
- [ ] Suspicious number of cases or deaths
- [ ] Missing data for a locality
- [ ] Missing time-series information
- [X] Other

I would appreciate returning the date format to yyyy/mm/dd or to ANY format other than yyyy-mm-dd as SPSS cannot read that format.  I am now forced to open the csv file in Notepad and change all the hypens to slashes so I can run my SPSS program.

**Comment by albertsun** [comment]
Hi @pnickelnm we've used this date format since the beginning and would not change it as that would cause more problems for most users.

I'm not familiar with SPSS specifically, but almost every statistical programming environment or analysis software should have some way of changing how it parses dates to accommodate for different date formats. I've heard good things about SPSS and would be checked if it didn't have this capability.

**Comment by pnickelnm** [comment]
Albert,

 

I apologize for the trouble post.  I do not know what was causing my problem yesterday, but it went away today.  I overwrite the file everyday so I could not compare the current file with earlier ones.

 

When I checked the SPSS manual it gave a lot of different formats for dates, but none for yyyy-mm-dd (year first, hyphens separating components).  I jumped to the conclusion that you had changed your format.  My code was reading it as what SPSS calls “SDate”.  Their manual lists SDate as looking for the year at the beginning of the string, but with “/” delimiting day, month, and year.  (Other formats are “Date” for dd-mmm-yyyy and “ADate” for mm/dd/yyyy.)

 

Yesterday when I ran my SPSS program it set all the dates to NULL.  I create counts and averages over 3-day and 7-day periods and need to verify that the record 6 prior to the current one is the same county and 6 days earlier.  I also use the dates to create a file for mapping which only includes data from “yesterday”.

 

I am totally baffled as to why the program worked for over 4 months and then did not yesterday, but worked today.  However, it is clear that the formatting of the date was NOT the problem.

 

Again, I apologize.

 

Phil Nickel
Community Empowerment Research
